src/components/contact-form/contact-form.jsx

```diff
@@ -0,0 +1,119 @@
+import { useForm } from 'react-hook-form';
+import styles from './contact-form.module.scss';
+import styledInput from '../shared/form-components/shared-input.module.scss';
+import EmailInput from '../shared/form-components/email-input';
+import DropDown from '../shared/form-components/drop-down';
+import InputShell from '../shared/form-components/input-shell';
+import PrimaryButton from '../shared/primary-button';
+
+export default function ContactForm() {
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors },
+  } = useForm();
+
+  const listItems = ['Landscape', 'Mountain', 'River', 'Grassland', 'House'];
+
+  const onSubmit = formData => {
+    console.log(formData);
+  };
+
+  return (
+    <div className={styles.container}>
+      <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+        <h1 className={styles.title}>Send a message</h1>
+
+        <InputShell
+          register={register}
+          name='firstName'
+          type='input'
+          required={true}
+          label={'First Name'}
+          classNames={{
+            gridArea: styles.firstName,
+          }}
+          errors={{
+            formError: errors.firstName,
+            message: 'First name is required',
+          }}
+        />
+
+        <InputShell
+          register={register}
+          name='lastName'
+          type='input'
+          required={true}
+          label={'Last Name'}
+          classNames={{
+            gridArea: styles.lastName,
+          }}
+          errors={{
+            formError: errors.lastName,
+            message: 'Last name is required',
+          }}
+        />
+
+        <EmailInput
+          classNames={{ email: styles.email }}
+          register={register}
+          errors={errors}
+        />
+
+        <DropDown
+          isForm={true}
+          listItems={listItems}
+          attributes={{
+            placeholder: 'Type painting',
+            ...register('paintings'),
+          }}
+          label={
```

Comment:
Sorry that's my bad, when I said to pass the label in as a prop, I meant to pass the text in as a prop - like how you did it for the input shell. Passing in a label element as a prop is a little odd.